### PR TITLE
144 refactor: replace all data uk icons

### LINKED ISSUE #144: Implement SVGs for the various precanned icons
Currently, this project is using UIKit v3 beta for a variety of purposes, including providing a rudimentary icon library. This library is no longer sufficient for our purposes and should be replaced with custom SVGs designed by @shanberg, which are currently located in `/src/images/icons/*.svg`

The goal of this project is to be able to treat our SVG icons as React components using the techniques described by https://css-tricks.com/creating-svg-icon-system-react/

Steps:
1. Refactor each of these SVGs into stateless functional React components
2. Enhance the React components to accept props for size and color
3. Enhance the React components with ARIA attributes as detailed in the above article and follow-on comments.
4. Render each of these icons in Storybook, demonstrating the possible uses of props.

### PR DESCRIPTION
**Pull request by karinakeen**
This PR resolves #144 

screen shot 2017-11-08 at 10 14 51 pm

@shanberg Is the cog glyph not a real icon for the settings button?
screen shot 2017-11-08 at 10 14 58 pm

### CODE REVIEW

**Comment by karinakeen** [comment]
This needs to be merged with PR 923 to pass circleci check.

File: src/components/Main/components/GMLineChart/index.js [hunk 1]
```diff
@@ -44,7 +48,10 @@ export default function GMLineChart({
         {timeSeries[0].length === 0 ? (
           <div className="chart-empty">
             <h1>
-              <i data-uk-icon={`icon: warning`} /> No Chartable Data
+              <Span>
+                <StatusIcon iconRatio={"1.8"} status={"warning"} />
```
Comment (shanberg) [inline src/components/Main/components/GMLineChart/index.js]: This shouldn't be connected to the StatusIcon. We should probably try to clarify that StatusIcon should only be used to indicate Service status.

```
<Icon backgroundStyle="BackgroundTriangleSmall">
  <Exclamation />
</Icon>
```
Something like this should give the same appearance.

File: src/components/Main/components/NotFoundError.js [hunk 2]
```diff
@@ -15,7 +17,9 @@ const NotFoundError = ({ errorMsg = "Not Found" }) => {
   return (
     <ErrorBox>
       <ErrorContent>
-        <Span data-uk-icon="icon: warning; ratio: 1.8" />
+        <Span>
+          <StatusIcon iconRatio={"1.8"} status={"warning"} />
```
Comment (shanberg) [inline src/components/Main/components/NotFoundError.js]: Same comment as in GMLineChart

File: src/components/Main/components/Settings/components/PollingSettings.js [hunk 3]
```diff
@@ -34,7 +32,7 @@ class PollingSettings extends Component {
     const buttonGlyph = isPolling ? "Pause" : "Play";
     const buttonLabel = isPolling ? "Pause Updates" : "Resume Updates";
     return (
-      <LayoutSection title={title} icon={PollIcon} flex>
+      <LayoutSection title={title} icon={"Poll"} flex>
```
Comment (shanberg) [inline src/components/Main/components/Settings/components/PollingSettings.js]: Can't tell you how happy this makes me.

File: src/components/Main/scenes/Instance/scenes/jvm/scenes/Threads/components/ThreadsSection/components/ThreadsTable/components/ThreadsTableLineItem/index.js [hunk 4]
```diff
@@ -86,10 +89,11 @@ export default class ThreadsTableLineItem extends Component {
         </TableColThread>
         <TableColThread>
           {stack.length ? (
-            <span
-              className="stack-trace-indicator icon"
-              data-uk-icon={"icon: table;"}
-            />
+            <span>
```
Comment (shanberg) [inline src/components/Main/scenes/Instance/scenes/jvm/scenes/Threads/components/ThreadsSection/components/ThreadsTable/components/ThreadsTableLineItem/index.js]: Can we get away with this without the `span`?

**Pull request merged**